.github/workflows/validate_cdk_release.yml

```diff
@@ -16,6 +16,12 @@ jobs:
   health_checks_with_cdk_version:
     uses: ./.github/workflows/health_checks.yml
     secrets: inherit
+    permissions:
```

Comment:
So this just runs all `jobs` in the `health_checks,yml` workflow? including things like [codeql](https://github.com/aws-amplify/amplify-backend/blob/main/.github/workflows/health_checks.yml#L946C3-L946C9)?